aura-cloud-dbaas/aura-grandstack/aura-grandstack.adoc

```diff
@@ -0,0 +1,321 @@
+= Deploying a GRANDstack application to Aura
+:slug: aura-grandstack
+:level: Intermediate
+:section: Neo4j Cloud DBaaS
+:section-link: aura-cloud-dbaas
+:sectanchors:
+:toc:
+:toc-title: Contents
+:toclevels: 1
+
+.Goals
+[abstract]
+In this page, we will learn how to build and deploy a GRANDstack application with Neo4j Aura.
+
+.Prerequisites
+[abstract]
+We should have created and started a Neo4j Aura database instance.
+If we have not already, we can follow the instructions in the link:https://aura.support.neo4j.com/hc/en-us/articles/360037562253-Working-with-Neo4j-Aura[getting started guide] to step through the process.
+We should also be familiar with the https://grandstack.io/[GRANDstack^].
+
+[role=expertise]
+{level}
+
+Now that we've created our Neo4j Aura database, we're going to learn how to deploy a GRANDstack application that queries data stored in that database.
+
+[#our-dataset]
+== The Football Transfers Graph
+
+We're going to deploy an application built on top of a graph containing football transfers scraped from https://www.transfermarkt.co.uk/[transfermarkt.co.uk].
+See https://medium.com/neo4j/football-transfers-graph-e8ba7347169e[Football Transfers Graph^] to learn more about the dataset.
+
+== Connect to Neo4j Aura with Neo4j Desktop
+
+The first thing we need to do, in link:/developer/neo4j-desktop/[Neo4j Desktop^], is create a remote connection to our Aura database.
+See the link:/developer/aura-connect-neo4j-desktop/[Connect to Neo4j Aura with Neo4j Desktop^] guide for step by step instructions explaining how to do this.
+
+[#neo4j-cloud-tool]
+== Neo4j Cloud Tool
+
+Now that we've got our Aura remote connection setup, we're going to import our sample dataset using the Neo4j Cloud Tool Graph App.
```

Comment:
This sentence is a bit confusing to me, as the next section is on import. Could we reword to say something like "we're going to use the Neo4j Cloud Tool to import our data, so let's see how to set up that application" or "we're going to set up the Neo4j Cloud Tool Graph app for importing our data".

Reply:
Have updated!